app/serializers/disability_claim_base_serializer.rb

```diff
@@ -69,6 +69,10 @@ def waiver_submitted
     requested_decision
   end
 
+  def claim_type
+    object_data['status_type']
```

Comment:
`status_type` isn't very descriptive here, the attribute is describing the claim and not a status. We could continue propagating the poor naming choice, but the possible confusion warrants renaming the attribute IMO